Feature new table (#21)
vtex-apps/paginated-table: docs/README.md

import { PersistedPaginatedTable } from 'vtex.paginated-table'
```

> If you want to use the table V2 and it's hooks do it as follows you can import only the hooks you will be using.

Reviewer: ```suggestion
If you want to use the table V2 and its hooks you can import just the hooks you will be using.
```
